cobalt/doc/setup-raspi.md

```diff
@@ -0,0 +1,245 @@
+# Set up your environment - Raspberry Pi
+
+These instructions explain how to set up Cobalt for your workstation and your
+Raspberry Pi device.
+
+## Set up your Raspberry Pi
+
+Download the latest Cobalt customized Raspbian image from <a
+href="https://storage.googleapis.com/cobalt-static-storage-public/2020-02-13-raspbian-buster-lite_shrunk_20210427.img">GCS bucket</a>
+(this is built via <a
+href="https://github.com/youtube/cobalt/tree/main/cobalt/tools/raspi_image">this
```

Comment:
This link doesn't exist but 
https://github.com/youtube/cobalt/tree/25.lts.1+/cobalt/tools/raspi_image
does.